packages/storybook/stories/va-file-input-uswds.stories.jsx

```diff
@@ -338,17 +350,21 @@ const PercentUploadedTemplate = args => {
       setPercent(_prev => {
         if (_prev >= 100) {
           clearInterval(intervalId);
-          return null;
+          return 100;
```

Comment:
Changing from `null` to `100` because the type checker expects this to always be a number. Felt 100 was safe since this is the percentage uploaded checker and we stop checking once it hits 100% or greater.